<launch>
    <arg name="cloud_topic" default="/camera/depth/color/points" />

    <node pkg="rail_segmentation" type="rail_segmentation" name="rail_segmentation" output="screen">
        <param name="zones_config" value="$(find fetch_grasp_suggestion)/config/zones.yaml" />
        <param name="point_cloud_topic" value="$(arg cloud_topic)" />
        <param name="max_cluster_size" value="20000" />
        <param name="min_cluster_size" value="75" />
        <param name="cluster_tolerance" value="0.02"/>
    </node>
    <!-- <node pkg="stretch_fetch_grasp_bridge" type="segmentation_dummy_node.py" name="modular_segmentation" output="screen"/> -->
    <node pkg="stretch_fetch_grasp_bridge" type="segmentation_node.py" name="modular_segmentation" output="screen"/>

</launch>